avocado/utils/network/interfaces.py

```diff
@@ -571,3 +571,29 @@ def restore_slave_cfg_file(self):
                     os.remove(slave_config)
             except Exception as ex:
                 raise NWException(f"Could not restore the config file {ex}")
+
+    def are_packets_lost(self, peer_ip, count, option=None, flood=False):
+        """
+        This method is used to check packet loss that occurs during
+        ping to peer address.
+
+        Function returns True for 0% packet loss and False
+        if packet loss occurs.
+
+        :param peer_ip: Peer IP address (IPv4 or IPv6)
+        :param count: How many packets to send
+        :param option: ping command options. Default is None
+        :param flood: ping flood option -f. Default is False
+        """
+        cmd = f'ping -I {self.name} {peer_ip} -c {count}'
```

Comment:
From the manpage:

"Only the super-user may use this option with zero interval."

So right now this is failing with a non root user. Because of that, combined with my other comments, I would recommend adding another argument called `sudo` to this method, where default is false. If the user is trying to do something powerful, then sudo must be true.

Reply:
Added one extra argument for super-users.
Sudo by default will be False, and if super-user option has to be used sudo must be provided as an aurgument.